src/js/script-loader.js

```diff
@@ -230,6 +230,33 @@ var LoaderProtoMethods = {
 						dependencies
 					);
 
+					self._log(
+						'Resolved modules:',
+						mappedModules,
+						'to:',
+						dependencies
+					);
+
+					var fastRejectError = '';
+
+					for (var i = 0; i < dependencies.length; i++) {
+						var dependency = dependencies[i];
+
+						if (dependency.indexOf(':ERROR:') == 0) {
+							fastRejectError += dependency.substr(7) + '\n';
+						}
+					}
+
+					if (fastRejectError.length > 0) {
+						reject(
+							new Error(
+								'The following problems where detected while ' +
+									'resolving modules:\n' +
+									fastRejectError
```

Comment:
It feels a bit weird that we're concatting errors and checking for string length here...

I'd prefer something like:

```javascript
const dependencyErrors = dependencies
    .filter(dep => dep.includes(':ERROR:'))
    .map(dep => dep.substr(7));

if (dependencyErrors.length) {
    reject(new Error(... + dependencyErrors.join('\n'));
}
```